### ISSUE TITLE
Unable to post nvda generated artifacts addons when forkers approved flows run

### ISSUE THREAD

**Issue by marlon-sousa** [marlon-sousa/EnhancedDictionaries#24]
As of today, approved flows by forkers are unable to generate the addons comments links due to token permission error.

We suggest changing comments generation to a flow which discovers the uploaded artifact (which is generated after last pr check) and comment a link for it. We also want one comment only to be updated.

**Comment by marlon-sousa** [comment]
@github-copilot run
